src/App.tsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
```

Comment:
In react whenever using scss or pure css you should be using [css modules](https://css-tricks.com/different-ways-to-write-css-in-react/#aa-use-css-modules) to ensure that every class name is scoped locally to the specific component in which it is being imported.